tools/launch_uv4.bat

```diff
@@ -0,0 +1,58 @@
+::
```

Comment:
Nitpick - might want to call this launch_uvision.bat rather than launch_uv4.  On my machine this starts up uvision5